package.json

```diff
@@ -13,10 +13,12 @@
     "@vitejs/plugin-react": "^1.3.2",
     "@vitejs/plugin-react-refresh": "^1.3.6",
     "antd": "^5.8.2",
+    "axios": "^1.7.8",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
     "react-icons": "^5.3.0",
     "react-router-dom": "^6.17.0",
+    "sweetalert2": "^11.14.5",
```

Comment:
Why do we need `sweetalert2`? We already have `AntD` installed. There is no point increasing dependecies on our project.

ref: https://ant.design/components/notification#notification-demo-with-icon